kubernetes/istio-rbac/istio-rbac.yaml

```diff
@@ -97,7 +97,22 @@ metadata:
   name: istio-sidecar-role-binding
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: details-service-account
```

Comment:
I don't think we can have the sample accounts in the 'default installation' instructions.